geode-core/src/main/java/org/apache/geode/cache/client/internal/ExecuteRegionFunctionOp.java

```diff
@@ -378,6 +379,22 @@ public ExecuteRegionFunctionOpImpl(String region, Function function,
       this.isHA = function.isHA();
     }
 
+    // For testing only
+    ExecuteRegionFunctionOpImpl() {
```

Comment:
Is there a way to test this without adding a new test hook to product code?

Reply:
I think the problem is that the product wasn't quite testable in the first place.  Yes we can mock around and use the existing constructors, but I think effort is better spent refactoring the class itself instead of munging tests.

Reply:
I'll work on the refactoring as a separate commit.  I think I'll modify the existing constructors (and new tests) to use a new constructor that assigns the final variables.